sdk/chainlink-solana/Cargo.toml

```diff
@@ -15,6 +15,9 @@ name = "chainlink_solana"
 default = []
 
 [dependencies]
-solana-program = "1.17.25"
-borsh = "0.9.1"
-borsh-derive = "0.9.1"
+solana-program = { version = "2.1.10" }
```

Comment:
One more tweak you might want to make, specifying `2.1` will make this crate work on any `2.x` Solana version from 2.1 onwards. That would make it work with 2.1 and 2.3

Reply:
We might have to do something for 3.x too: https://docs.rs/solana-program/3.0.0/solana_program/index.html